typography/typography.css

```diff
@@ -0,0 +1,62 @@
+/* Section container for typography items */
+.disp-typo {
+  display: flex;
+  padding: 0 20rem;
+  flex-direction: column;
+  text-align: left;
+  margin: 1rem;
+}
+
+.sub-title {
+  font-size: 1.2rem;
+}
+
+.h-1 {
+  font-size: 48px;
```

Comment:
you can change these to rems so it will also change according to screens.

Reply:
Done, man!